### PR TITLE
[WIP] First pass at migrating logging puts to ohai/opoo, etc.

### PR DESCRIPTION
**Pull request by colindean**
Work toward #1202, but untested. I did this commit in a codespace
that is flailing.

### CODE REVIEW

File: cmd/bundle.rb [hunk 1]
```diff
@@ -162,22 +162,21 @@ def bundle
       end
     rescue SystemExit => e
       Homebrew.failed = true unless e.success?
-      puts "Kernel.exit" if args.debug?
+      odebug "Kernel.exit" if args.debug?
```
Comment (MikeMcQuaid) [inline cmd/bundle.rb:165]:
```suggestion
      odebug "Kernel.exit"
```

may work as expected, worth a try.

File: cmd/bundle.rb [hunk 2]
```diff
@@ -162,22 +162,21 @@ def bundle
...
     rescue Interrupt
-      puts # seemingly a newline is typical
-      Homebrew.failed = true
+      ofail "Interrupted"
```
Comment (MikeMcQuaid) [inline cmd/bundle.rb:167]: Don't do this

File: cmd/bundle.rb [hunk 3]
```diff
@@ -162,22 +162,21 @@ def bundle
...
     rescue RuntimeError, SystemCallError => e
       raise if e.message.empty?
 
       onoe e
-      puts e.backtrace if args.debug?
-      Homebrew.failed = true
+      ofail e.backtrace if args.debug?
```
Comment (MikeMcQuaid) [inline cmd/bundle.rb:172]: Don't want to do this, it makes the output worse for no real reason

File: cmd/bundle.rb [hunk 4]
```diff
@@ -162,22 +162,21 @@ def bundle
...
     rescue => e
       onoe e
-      puts "#{Tty.bold}Please report this bug:#{Tty.reset}"
-      puts "  #{Formatter.url("https://github.com/Homebrew/homebrew-bundle/issues")}"
-      puts e.backtrace
-      Homebrew.failed = true
+      ohai(
```
Comment (MikeMcQuaid) [inline cmd/bundle.rb:175]: Makes output worse

**Review by MikeMcQuaid — CHANGES_REQUESTED** [review]
As said in other issue: most of these changes don't make sense. `odebug` might do but all `ohai` and many `opoo` etc. use makes output worse, sorry.

**Comment by colindean** [comment]
I'll revisit this less hurriedly and haphazardly, and with actual testing, in the future. Devcontainers setup should make it easier to do meaningful drive-by contributions like this.

**Pull request closed without merging**